src/data/gallery/pictures.js

```diff
@@ -0,0 +1,89 @@
+import fall235 from "@/public/images/fall/fall23_5.webp";
+import fall236 from "@/public/images/fall/fall23_6.webp";
+import fall238 from "@/public/images/fall/fall23_8.webp";
+import fall239 from "@/public/images/fall/fall23_9.webp";
+import fall2310 from "@/public/images/fall/fall23_10.webp";
+import fall2311 from "@/public/images/fall/fall23_11.webp";
+import fall2312 from "@/public/images/fall/fall23_12.webp";
+import fall2313 from "@/public/images/fall/fall23_13.webp";
+import fall2314 from "@/public/images/fall/fall23_14.webp";
+import fall2315 from "@/public/images/fall/fall23_15.webp";
+import fall2316 from "@/public/images/fall/fall23_16.webp";
+import fall2317 from "@/public/images/fall/fall23_17.webp";
+import fall2318 from "@/public/images/fall/fall23_18.webp";
+import fall2319 from "@/public/images/fall/fall23_19.webp";
+import fall2320 from "@/public/images/fall/fall23_20.webp";
+import fall2321 from "@/public/images/fall/fall23_21.webp";
+import fall2322 from "@/public/images/fall/fall23_22.webp";
+import fall2323 from "@/public/images/fall/fall23_23.webp";
+import fall2324 from "@/public/images/fall/fall23_24.webp";
+import winter241 from "@/public/images/winter/winter24_1.webp";
+import winter242 from "@/public/images/winter/winter24_2.webp";
+import winter243 from "@/public/images/winter/winter24_3.webp";
+import winter244 from "@/public/images/winter/winter24_4.webp";
+import winter245 from "@/public/images/winter/winter24_5.webp";
+import winter246 from "@/public/images/winter/winter24_6.webp";
+import winter247 from "@/public/images/winter/winter24_7.webp";
+import winter248 from "@/public/images/winter/winter24_8.webp";
+import spring241 from "@/public/images/spring/spring24_1.webp";
+import spring242 from "@/public/images/spring/spring24_2.webp";
+import spring243 from "@/public/images/spring/spring24_3.webp";
+import spring244 from "@/public/images/spring/spring24_4.webp";
+import spring245 from "@/public/images/spring/spring24_5.webp";
+import spring246 from "@/public/images/spring/spring24_6.webp";
+import spring247 from "@/public/images/spring/spring24_7.webp";
+import spring248 from "@/public/images/spring/spring24_8.webp";
+import spring249 from "@/public/images/spring/spring24_9.webp";
+
+export const PICTURES = {
+  fall: [
+    fall231,
+    fall232,
+    fall233,
+    fall234,
+    fall235,
+    fall236,
+    fall238,
+    fall239,
+    fall2310,
+    fall2311,
+    fall2312,
+    fall2313,
+    fall2314,
+    fall2315,
+    fall2316,
+    fall2317,
+    fall2318,
+    fall2319,
+    fall2320,
+    fall2321,
+    fall2322,
+    fall2323,
+    fall2324,
+  ],
+  winter: [
+    winter241,
+    winter242,
+    winter243,
+    winter244,
+    winter245,
+    winter246,
+    winter247,
+    winter248,
+  ],
+  spring: [
+    spring241,
+    spring242,
+    spring243,
+    spring244,
+    spring245,
+    spring246,
+    spring247,
+    spring248,
+    spring249,
+  ],
```

Comment:
need one more page for past events.

you can find the pictures under /public/images/past